### PR TITLE
Request resize

### PR DESCRIPTION
**Pull request by juliareichel**
Fixed requestResize function to take in parameters that match those of the spec

### CODE REVIEW

File: examples/player/simid_player.js [hunk 1]
```diff
@@ -289,10 +289,10 @@ class SimidPlayer {
       return;
     }
 
-    this.simidIframe_.style.height = incomingMessage.args['height'];
-    this.simidIframe_.style.width = incomingMessage.args['width'];
-    this.simidIframe_.style.left = incomingMessage.args['x_val'];
-    this.simidIframe_.style.top = incomingMessage.args['y_val'];
+    this.simidIframe_.style.height = incomingMessage.args.creativeDimensions['height'];
+    this.simidIframe_.style.width = incomingMessage.args.creativeDimensions['width'];
+    this.simidIframe_.style.left = incomingMessage.args.creativeDimensions['x_val'];
```
Comment (damargulis) [inline examples/player/simid_player.js:294]: should be `x` instead of `x_val`?

Comment (juliareichel) [reply examples/player/simid_player.js]: good spot

File: examples/player/simid_player.js [hunk 2]
```diff
@@ -289,10 +289,10 @@ class SimidPlayer {
       return;
     }
 
-    this.simidIframe_.style.height = incomingMessage.args['height'];
-    this.simidIframe_.style.width = incomingMessage.args['width'];
-    this.simidIframe_.style.left = incomingMessage.args['x_val'];
-    this.simidIframe_.style.top = incomingMessage.args['y_val'];
+    this.simidIframe_.style.height = incomingMessage.args.creativeDimensions['height'];
```
Comment (ryanthompson591) [inline examples/player/simid_player.js:292]: Can you add a TODO to make a separate PR that validates the parameters sent it.  It should reject (with an appropriate error message) when the parameters are larger than the player size.

Comment (Kaleab470) [reply examples/player/simid_player.js:292]: I have taken care of it in my PR
